helm/cluster-gcp/templates/_machine_deployments.tpl

```diff
@@ -53,6 +53,9 @@ spec:
       image: {{ include "vmImage" $global }}
       instanceType: {{ .instanceType | default "n2-standard-4" }}
       rootDeviceSize: {{ .rootVolumeSizeGB | default 100 }}
+      serviceAccounts:
```

Comment:
just wondering why this says `serviceAccounts`  like in plural if there is no array but simple object with option only for serviceAccount
 
 is that a typo on our side or CAPG being stupid?

Reply:
I think it's CAPG being stupid. It's defined as an object
```
k --kubeconfig=./kubeconfigs/frieza.kubeconfig explain gcpmachinetemplates.spec.template.spec.serviceAccounts                                 [gs-gunther:default]
KIND:     GCPMachineTemplate
VERSION:  infrastructure.cluster.x-k8s.io/v1beta1

RESOURCE: serviceAccounts <Object>

DESCRIPTION:
     ServiceAccount specifies the service account email and which scopes to
     assign to the machine. Defaults to: email: "default", scope:
     []{compute.CloudPlatformScope}

FIELDS:
   email        <string>
     Email: Email address of the service account.

   scopes       <[]string>
     Scopes: The list of scopes to be made available for this service account.
```